data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
```

Comment:
As far as minimum crew goes, max bunk space should probably be considered as well. Right now, this "converted colony ship" has room for 170 non-essential personnel, and it manages this with a better crew-to-passenger ratio than the next largest "passenger" ship, the Star Queen. Now yes, the Star Queen is a luxury vessel -  but it's also not dual-hatted as a combat ship/flagship and it's presumably hauling people with less baggage for significantly shorter periods of time. Even after being repurposed, it could (I'd say would) make sense to move the Bac's ~29% crew-to-bunks ratio a bit closer to the SQ's 38% ratio.

Also, if it's older, presumably it has less advanced core systems, less automation, and would require more people to do the same work than a modern heavy warship. I think what makes it tempting to use warships as the model for crew size is the lack of other big civilian ships, except freighters - and freighters are a special case with lot of empty space compared to functional or habitable ships.

The other thing is that having relatively low min crew to bunk space means you can cap a lot more and bigger ships before you take a performance hit. That seems a lot more like something a military ship should be able to do than a civilian one...